mmdet3d/datasets/nuscenes_mono_dataset.py

```diff
@@ -0,0 +1,665 @@
+import copy
+import mmcv
+import numpy as np
+import pyquaternion
+import tempfile
+import torch
+from nuscenes.utils.data_classes import Box as NuScenesBox
+from os import path as osp
+
+from mmdet3d.core import bbox3d2result, box3d_multiclass_nms, xywhr2xyxyr
+from mmdet.datasets import DATASETS
+from ..core.bbox import CameraInstance3DBoxes, get_box_type
+
+try:
```

Comment:
Move coco to where it is imported, see https://github.com/open-mmlab/mmdetection/pull/4880